### DISCUSSION TITLE
MacOS settings

### DISCUSSION THREAD

**Discussion by Elegusch** [xvzc/SpoofDPI#280] [Show and tell]
Spoofer hasn't been working for 2 weeks, today began under 
spoof-dpi --enable-doh --window-size 0

MacOS Ventura 13.7

**Comment by intelligence67** [comment]
Can you describe the whole path?

I launch a terminal, paste the command curl -fsSL https://raw.githubusercontent.com/xvzc/SpoofDPI/main/install.sh | bash -s darwin-arm64 . After it I write export PATH=$PATH:~/.spoofdpi/bin and then enter your command, but it doesn’t work for me. And if I just enter spoofdpi , it starts.

**Reply by Elegusch** [reply]
same steps, but I use .sh script(pic attached) to do this on login into system, of on double click from desktop. Now when im home on Rostelecom it doesn't work, but at my aftergrad's public wifi worked today. On the side note I have an Intel MacBook
<img width="794" alt="Screenshot 2024-10-24 at 21 32 27" src="https://github.com/user-attachments/assets/536536d8-90c2-4cf4-9db8-4344beced8ca">
